students/brgalloway/Lesson_6/test_mailroom_part_4.py

```diff
@@ -0,0 +1,71 @@
+from mailroom_part_4 import *
+import os
+
+a_dict = {
+        "Jeff Bezos": {
+            "donation_total": 877.33,
+            "times_donated": 1,
+            "average_donation": 877.33
+        },
+        "Paul Allen": {
+            "donation_total": 708.42,
+            "times_donated": 3,
+            "average_donation": 236.14
+        }
+    }
+
+'''
+ensure that list of names returned is a string
+'''
+def test_list_names():
+    names = list_names()
+    assert type(names) is str
+
+'''
+Verify sorted donors is always returned as a list
+'''
+def test_sort_donors():
+    output = sorted(a_dict.items(), key=sort_donors, reverse=True)
+    assert type(output) is list
```

Comment:
instead of checking a list check for contents of a list, for example:
`assert output == ['Alex', 'Bob', 'Trisha']`